.azure-pipelines/OfficialBuild.yml

```diff
@@ -34,9 +34,9 @@ steps:
   displayName: Capture build version in package/version.csv
 
 - task: NodeTool@0
-  displayName: 'Use nodejs 14.x'
+  displayName: 'Use nodejs 16.x'
   inputs:
-    versionSpec: '14.x'
+    versionSpec: '16.x'
```

Comment:
nodejs 16 (or even 18, the new LTS) is fine as toolchaing to >build< the extension; near term we should change our PR workflow to run our VSCode extension tests on either the min bar version of VSCode (14??), or run tests on a matrix of former and current LTS, and 19?

Reply:
Created a work item to track testing/upgrading to LTS 18